packages/opentelemetry-metrics/src/MeterProvider.ts

```diff
@@ -49,8 +49,8 @@ export class MeterProvider implements api.MeterProvider {
    *
    * @returns Meter A Meter with the given name and version
    */
-  getMeter(name: string, version = '*', config?: MeterConfig): Meter {
-    const key = `${name}@${version}`;
+  getMeter(name: string, version?: string, config?: MeterConfig): Meter {
+    const key = `${name}@${version || ''}`;
```

Comment:
Should the @ be left out here too if the version is empty?

Reply:
The key name is an implementation detail so it's don't care.